app/dashboard/utils.py

```diff
@@ -186,6 +188,9 @@ def getBountyContract(network):
 
 
 def get_bounty(bounty_enum, network):
+    if (settings.DEBUG or settings.ENV != 'prod') and network == 'mainnet':
```

Comment:
should this b e in the PR?

Reply:
Yeah. I intentionally added this to avoid even syncing `mainnet` bounties on local.